subprojects/core-api/src/main/java/org/gradle/api/plugins/ObjectConfigurationAction.java

```diff
@@ -1,73 +1,84 @@
-/*

```

Comment:
What the heck did I do do this file to cause this diff??

Reply:
Line ending changes (from \r\n to \n).

And thank you for pushing HTTPS usage :)

Reply:
Oh. Thanks for letting me know it was a line ending thing.

Reply:
Wasn't the line-ending fixes PR merged a long time ago?

Reply:
Yea, it was. This PR was the reason why I ended up needing to do the line ending PR.

I got the line ending PR merged before this one.